### PR TITLE
Enable GitHub actions, discard Travis builds

### PR DESCRIPTION
**Pull request by andreibancioiu**
- Remove Travis build.
 - Existing workflow `Build and Deploy Testnet` - only trigger explicitly.
 - New workflow `Build`.
 - New workflow `Create release` (when pushing to master / main) Will create a draft release with the assets. Also available to be triggered explicitly, at any given moment (specifying the branch is necessary).

How to trigger explicitly: from [the user interface](https://github.com/ElrondNetwork/elrond-go/actions). But buttons will only be enabled once the PR reaches the main branch (master).

### CODE REVIEW

File: process/factory/shard/vmContainerFactory.go [hunk 1]
```diff
@@ -79,9 +79,9 @@ func (vmf *vmContainerFactory) Create() (process.VirtualMachinesContainer, error
 }
 
 func (vmf *vmContainerFactory) createArwenVM() (vmcommon.VMExecutionHandler, error) {
-	if vmf.config.OutOfProcessEnabled {
-		return vmf.createOutOfProcessArwenVM()
-	}
+	// if vmf.config.OutOfProcessEnabled {
```
Comment (iulianpascalau) [inline process/factory/shard/vmContainerFactory.go:82]: commented code - without TODO

Comment (andreibancioiu) [reply process/factory/shard/vmContainerFactory.go]: Fixed (was for debugging the MacOS build).

File: .github/workflows/build_and_test.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      - name: Build
+        run: |
+          export ARWEN_PATH=${GITHUB_WORKSPACE}/arwen
+          cd ${GITHUB_WORKSPACE}/cmd/node && go build .
+          cd ${GITHUB_WORKSPACE}/cmd/keygenerator && go build .
+          cd ${GITHUB_WORKSPACE}/cmd/logviewer && go build .
+          cd ${GITHUB_WORKSPACE}/cmd/termui && go build .
+          cd ${GITHUB_WORKSPACE} && make arwen
+
+      # Skip long-running and other intensive tests
```
Comment (iulianpascalau) [inline .github/workflows/build_and_test.yml:43]: Please consider removing this

Comment (andreibancioiu) [reply .github/workflows/build_and_test.yml]: Fixed (removed).

**Review by iulianpascalau — DISMISSED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]

**Review by LucianMincu — APPROVED** [review]
👍

**Pull request merged**